tests/test_state/test_storage_state.py

```diff
@@ -595,4 +595,25 @@ def test_to_dict_keys_in_return_dict(self):
         storage_state = StorageState()
         current_time_slot, _, _ = self._initialize_time_slots()
         storage_state.add_default_values_to_state_profiles([current_time_slot])
-        assert set(storage_state.to_dict(current_time_slot).keys()).issubset(SAMPLE_STATS.keys())
+        storage_state.energy_to_sell_dict[current_time_slot] = "test_energy_to_sell"
+        storage_state.offered_sell_kWh[current_time_slot] = "test_energy_active_in_bids"
+        storage_state.energy_to_buy_dict[current_time_slot] = "test_energy_to_buy"
+        storage_state.offered_buy_kWh[current_time_slot] = "test_energy_active_in_offers"
```

Comment:
Any reason why you did not use floats here? Is the a common practice for tests?

Reply:
There is no specific reason behind these other than I found somehow arbitrary to set a specific value to make the assertion with. I've seen this strategy in some testing on online forums, though. I'd say this is just a mater of personal taste. Any float-str value would have make the job.